src/evaluate/evaluator/token_classification.py

```diff
@@ -0,0 +1,295 @@
+
+                - `"simple"` - we evaluate the metric and return the scores.
+                - `"bootstrap"` - on top of computing the metric scores, we calculate the confidence interval for each
+                of the returned metric keys, using `scipy`'s `bootstrap` method
+                https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html.
+            confidence_level (`float`, defaults to `0.95`):
+                The `confidence_level` value passed to `bootstrap` if `"bootstrap"` strategy is chosen.
+            n_resamples (`int`, defaults to `9999`):
+                The `n_resamples` value passed to `bootstrap` if `"bootstrap"` strategy is chosen.
+            random_state (`int`, *optional*, defaults to `None`):
+                The `random_state` value passed to `bootstrap` if `"bootstrap"` strategy is chosen. Useful for
+                debugging.
+            input_column (`str`, defaults to `"tokens"`):
+                the name of the column containing the tokens feature in the dataset specified by `data`.
+            label_column (`str`, defaults to `"label"`):
+                the name of the column containing the labels in the dataset specified by `data`.
+            label_mapping (`Dict`, *optional*, defaults to `None`):
+                We want to map class labels defined by the model in the pipeline to values consistent with those
+                defined in the `label_column` of the `data` dataset.
+            join_by (`str`, *optional*, defaults to `" "`):
+                This evaluator supports dataset whose input column is a list of words. This parameter specifies how to join
+                words to generate a string input. This is especially useful for languages that do not separate words by a space.
+
+        Return:
+            A `Dict`. The keys represent metric keys calculated for the `metric` spefied in function arguments. For the
+            `"simple"` strategy, the value is the metric score. For the `"bootstrap"` strategy, the value is a `Dict`
+            containing the score, the confidence interval and the standard error calculated for each metric key.
+
+        The dataset input and label columns are expected to be formatted as a list of words and a list of labels respectively, following [conll2003 dataset](https://huggingface.co/datasets/conll2003). Datasets whose inputs are single strings, and labels are a list of offset are not supported.
+
+        Examples:
+        ```python
+        >>> from evaluate import evaluator
+        >>> from datasets import load_dataset
+        >>> task_evaluator = evaluator("token-classification")
+        >>> data = load_dataset("conll2003", split="validation[:2]")
+        >>> results = task_evaluator.compute(
+        >>>     model_or_pipeline="elastic/distilbert-base-uncased-finetuned-conll03-english",
+        >>>     data=data,
+        >>>     metric="seqeval",
+        >>>     join_by=" ",
```

Comment:
I guess this one is not necessary, right? since it's the default.
```suggestion
```

Reply:
I left it to make it explicit we were joining the strings in the list. But I agree it can be removed.